packages/std/src/amp/messages.rs

```diff
@@ -304,6 +304,7 @@ pub struct AMPPkt {
     /// Any messages associated with the packet
     pub messages: Vec<AMPMsg>,
     pub ctx: AMPCtx,
+    pub id: Option<String>,
```

Comment:
We shouldn't need this anymore, right?

Reply:
It's being used here:
```rust
let id = match packet.id.clone() {
        // Generate unique ID if the packet doesn't already have one
        Some(id) => validate_id(&id, &env.block.chain_id, env.block.height, tx_index)?,
        // Not using "-" as a separator since chain id can contain it
        None => format!("{}.{}.{}", env.block.chain_id, env.block.height, tx_index),
    };
```

Reply:
Right but isn't the packet ID under `AMPCtx` now?